tiatoolbox/tools/tissuemask.py

```diff
@@ -17,6 +17,7 @@ class TissueMasker(ABC):
     """
 
     def __init__(self) -> None:
+        """Initializes :class:`TissueMasker`."""
```

Comment:
```suggestion
        """Initialize :class:`TissueMasker`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090